src/controllers/dynakube/version/activegate.go

```diff
@@ -4,19 +4,19 @@ import (
 	"context"
 
 	"github.com/Dynatrace/dynatrace-operator/src/api/status"
-	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube"
-	"github.com/Dynatrace/dynatrace-operator/src/dockerconfig"
+	"github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube"
```

Comment:
I think following the pattern of `&dynatracev1beta1.DynaKube{}` would be preferred. (using the actual api version for clarity)
also you would be avoiding shadowing the package

Reply:
[Done](https://github.com/Dynatrace/dynatrace-operator/pull/1998/commits/56696004da9626423e3e56b78989950e4b3db933)